packages/voyager-metrics/src/metrics.ts

```diff
@@ -72,16 +92,31 @@ export function updateResolverMetrics (resolverInfo: any, responseTime: number)
   resolverRequestsMetric
     .labels(
       resolverMappingType,
-      path
+      path,
+      success + '',
```

Comment:
Question: Is this hack to convert the value to a String?

Reply:
Yes otherwise it won't compile

Reply:
I saw this on SO: https://stackoverflow.com/questions/51882134/prometheus-query-to-count-unique-label-values. So I think if we just sent a counter metric data with the label: `clientId=<value>`, and the we can just count it like this:

```
count( count by (clientId) (unique_clients))
```